torch/quantization/__init__.py

```diff
@@ -12,6 +13,23 @@ def default_eval_fn(model, calib_data):
     for data in calib_data:
         model(data)
 
+default_loss_fn = torch.nn.MSELoss(reduction='sum')
+def default_train_fn(model, train_data, loss_fn=default_loss_fn, optimizer=None):
+    r"""
+    Default train function takes a torch.utils.data.Dataset and train the model
+    on the dataset
+    """
+    if not optimizer:
```

Comment:
it's pretty sketchy to do it

Reply:
removed, and the function is moved to common_quantization.py